packages/frontend/src/components/Game/GameBoard/TileGrid/style.scss

```diff
@@ -0,0 +1,11 @@
+
+
+.tile_grid {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    
+}
+.game_row {
+    display: flex;
+}
```

Comment:
These can all be set with bootstrap classes: `d-flex flex-column align-items-center`